extensions/birthday.py

```diff
@@ -9,31 +9,43 @@ class BoostersExtension(dippy.Extension):
 
     def __init__(self):
         super().__init__()
-        self.when = datetime(2023, 11, 13, 1, 0, 0, 0)
+        now = datetime.utcnow()
+        self.when = datetime(now.year, 11, 13, 1, 0, 0, 0)
         self.setup()
 
-    def setup(self):
+    def update_next_birthday(self):
         if datetime.utcnow() > self.when:
-            return
+            self.when = self.when.replace(year=self.when.year + 1)
+
+    def how_many_years_old(self) -> str:
+        years = self.when.year - 2020
+        if years > 3:
```

Comment:
What? You think we're not gonna be around for our 21st year 😂

Reply:
I missed that edge case LOL. Well, at-least we got 18 years to solve this bug! 😂